src/DDLQueryBuilder.php

```diff
@@ -171,23 +172,20 @@ public function renameColumn(string $table, string $oldName, string $newName): s
      *
      * @return string The column definition.
      */
-    public function getColumnDefinition(string $table, string $column): string
+    private function getColumnDefinition(string $table, string $column): string
     {
-        $result = '';
         $sql = $this->schema->getTableSchema($table)?->getCreateSql();
 
         if (empty($sql)) {
             return '';
         }
 
-        if (preg_match_all('/^\s*([`"])(.*?)\\1\s+(.*?),?$/m', $sql, $matches) > 0) {
-            foreach ($matches[2] as $i => $c) {
-                if ($c === $column) {
-                    $result = $matches[3][$i];
-                }
-            }
+        $quotedColumn = preg_quote($column, '/');
+
+        if (preg_match("/^\s*([`\"])$quotedColumn\\1\s+(.*?),?$/m", $sql, $matches) !== 1) {
+            return '';
```

Comment:
Need update phdoc (about returns empty string and throws Throwable)

Reply:
Done